plugins/node/opentelemetry-instrumentation-fastify/README.md

```diff
@@ -45,6 +45,28 @@ registerInstrumentations({
 
 See [examples/fastify](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/fastify) for a short example.
 
+## Fastify Instrumentation Options
+
+| Options | Type | Example | Description |
+| `requestHook` | `FastifyCustomAttributeFunction` | `(span, request) => {}` | Function for adding custom attributes to Fastify requests. Receives params: `Span, FastifyRequest`. |
```

Comment:
update to info here as well: `(span, requestInfo)`, Receives params: `Span, FastifyRequestInfo`